Add pages to create and view experiments

niklhut opened the pull request:
Adds a page to edit experiments and fixes #222. Also links the newest experiments in the profile page, therefore a get own experiments endpoint was created, fixes #210. Finally, a detail page for experiments was added, fixes #259. Also fixed some problems with the experiment overview page.

niklhut commented:
Now also fixes #278

LeviAchenbach commented:
using the create experiment button while the user is not verified will lead to an error page like "please verify your email first, before creating experiments" but an empty experiment will be createt nonetheless

LeviAchenbach commented:
I dont know if this is to be solved in this issue, but uploading does not work yet (you can select/drag an drop files but nothing will be actually visible in the experiment except the text fields)

niklhut commented:
Thanks, the ability to create experiments was not correct, i fixed this. Also I added a text describing that an unverified user needs to verify their mail first before creating new experiments. 
Regarding the files, they already work but you need to add the `NUXT_FILE_MOUNT=./public` variable to you .env

Tim-Kroker commented:
Except for the Attribute "Vorbereitungszeit" one should be able to associate multiple values of one attribute with an experiment which currently is not supported by the UI

suiluj482 reviewed (approved):
lgtm
